src/Interfaces/ICurve.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.17;
+
+/// @title Interface for Curve
+/// @author LI.FI (https://li.fi)
+/// @custom:version 1.0.0
+interface ICurve {
+    function exchange(
+        int128 i,
+        int128 j,
+        uint256 dx,
+        // solhint-disable-next-line var-name-mixedcase
```

Comment:
could we disable solhint for interface folder? 
Would imho make sense..

Reply:
good point!